json/Item_Weapon_Compoundbow.txt

```diff
@@ -618,7 +618,7 @@
 	{
 		"assign": "1150148",
 		"jp_text": "白雉春光",
-		"tr_text": "Shirokiji Shunkou",
+		"tr_text": "Wakuchi Shunkou",
```

Comment:
According to http://pso2.swiki.jp/index.php?%E7%99%BD%E9%9B%89%E6%98%A5%E5%85%89 the reading is "Hakuchi Shunkou", but I don't know if they are any more authoritative than where you got your reading from.